app/email/service.go

```diff
@@ -143,6 +143,15 @@ type ServiceInterface interface {
 	SendMailWithEmbeddedFiles(to, subject, htmlBody string, embeddedFiles map[string]io.Reader, messageID string, inReplyTo string, references string) error
 	SendLicenseUpForRenewalEmail(email, name, locale, siteURL, renewalLink string, daysToExpiration int) error
 	SendPaymentFailedEmail(email string, locale string, failedPayment *model.FailedPayment, siteURL string) (bool, error)
+	// Group into 1 function?? Idk..
```

Comment:
Would be a good idea to reduce noise here at this level but at the same time there are several params that are different.